.github/ISSUE_TEMPLATE

```diff
@@ -1,17 +1,45 @@
 提交 Issue 之前请先阅读 [Issue 指引](https://www.v2ray.com/zh_cn/chapter_01/issue.html)，然后回答下面的问题，谢谢。
 Please answer the following questions before submitting your issue. Thank you.
 
-1) 你正在使用哪个版本的 V2Ray？（如果服务器和客户端使用了不同版本，请注明）
-1) What version of V2Ray are you using?
+1.  你正在使用哪个版本的 V2Ray？（如果服务器和客户端使用了不同版本，请注明）
+    Which version of V2Ray are you using(If you deploy different version on server and client, please explicitly point out)?
 
-2) 你的使用场景是什么？比如使用 Chrome 通过 Socks/VMess 代理观看 YouTube 视频。
-2) What your scenario of using V2Ray? E.g., Watching YouTube videos in Chrome via Socks/VMess proxy.
+2)  你的使用场景是什么？比如使用 Chrome 通过 Socks/VMess 代理观看 YouTube 视频。
+    What's your scenario of using V2Ray? E.g., Watching YouTube videos in Chrome via Socks/VMess proxy.
 
-3) 你看到的不正常的现象是什么？
-3) What did you see?
+3)  你看到的不正常的现象是什么？
+    What did you see?
 
-4) 你期待看到的正确表现是怎样的？
-4) What do you expected to see instead?
+4)  你期待看到的正确表现是怎样的？
+    What's your expection?
 
-5) 请附上你的配置文件。
-5) Please attach your configuration file.
+5)  请附上你的配置文件（请注意隐藏服务器端IP地址与UUID）。
+    Please attach your configuration file(HIDE YOUR SERVER IP ADDRESS AND UUID BEFORE SUBMIT ISSUE!).
+
+    Server Configuration File（服务器端配置文件）:
+    ```
+    // 在这里附上服务器端配置文件
+    // Please attach your server configuration file here.
+    ```
+    
+    Client Configuration File（客户端配置文件）:
+    ```
+    // 在这里附上客户端配置文件
+    // Please attach your client configuration file here.
+    ```
+    
+6)  请附上出错时软件输出的日志。
+    Please attach log file thrown by software when something goes wrong.
```

Comment:
Please attach the log file, especially the bottom lines if the file is large. Log file is usually `/var/log/v2ray/error.log` on Linux.